tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -21,8 +21,8 @@ steps:
     --enable_training \
     --mpi_home /home/ciagent/pkg/openmpi-4.0.5 \
     --use_rocm \
-      --rocm_home /opt/rocm-3.9.0 \
-      --nccl_home /opt/rocm-3.9.0 \
+      --rocm_home /opt/rocm \
```

Comment:
The indentation can be better.

Reply:
OK, yeah. It was sub-indent for rocm options.

Reply:
FWIW, the indentation made sense to me. could also put the related options on the same line.